src/react-components/package.json

```diff
@@ -17,7 +17,7 @@
     "dist": "webpack --mode production --progress --display-modules --config webpack.prod.js",
     "watch-webpack": "webpack --mode development --watch --progress --display-modules --config webpack.prod.js",
     "lint": "tslint -c tslint.json src/__tests__/components/*.tsx src/components/**/**/*.tsx src/components/**/**/**/*.tsx src/utils/*.tsx",
-    "test": "rm -rf lib && npm run lint && tsc && mocha --require lib/__tests__/*.js lib/__tests__/**/*.js"
+    "test": "rm -rf lib && npm run lint && tsc && mocha --require lib/__tests__/*.js lib/__tests__/**/*.js lib/__tests__/**/**/*.js"
```

Comment:
Can we use `--recursive` here so we're not continuing to pile stuff on?  
https://mochajs.org/#usage

Reply:
This is great.